Subject: [PATCH] m68k: Export ARAnyM native feature API

From: Geert Uytterhoeven <geert@linux-m68k.org>

ERROR: "nf_get_id" [arch/m68k/emu/nfeth.ko] undefined!
ERROR: "nf_call" [arch/m68k/emu/nfeth.ko] undefined!
ERROR: "nf_get_id" [arch/m68k/emu/nfcon.ko] undefined!
ERROR: "nf_call" [arch/m68k/emu/nfcon.ko] undefined!
ERROR: "nf_call" [arch/m68k/emu/nfblock.ko] undefined!
ERROR: "nf_get_id" [arch/m68k/emu/nfblock.ko] undefined!

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 arch/m68k/emu/natfeat.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/arch/m68k/emu/natfeat.c
+++ b/arch/m68k/emu/natfeat.c
@@ -13,6 +13,7 @@
 #include <linux/console.h>
 #include <linux/string.h>
 #include <linux/kernel.h>
+#include <linux/module.h>
 #include <linux/io.h>
 #include <asm/machdep.h>
 #include <asm/natfeat.h>
@@ -31,6 +32,8 @@ asm("\n"
 "	.long	nf_get_id,1b\n"
 "	.long	nf_call,1b\n"
 "	.previous");
+EXPORT_SYMBOL_GPL(nf_get_id);
+EXPORT_SYMBOL_GPL(nf_call);
 
 static int stderr_id;
 
